pacmanfile

```diff
@@ -29,6 +29,11 @@ fi
 config_dir=${XDG_CONFIG_HOME:-$HOME/.config}
 pacmanfile_dir="${config_dir}/pacmanfile"
 
+# Create config folder if not present
```

Comment:
Thanks, @O2-AC! I think this is a great addition. However, I would place this block in the `dump` function (on the current line 64) because the creation of the directory is unexpected to me if, for example, I just run the `help` command. So I would only run the check when the `dump` command is actually ran.

Reply:
That is probably the better place for that sanity check, I agree. I have added the changes in the new commit.